Meadow.FSharp.Template/Program.fs

```diff
@@ -1,40 +1,55 @@
+namespace MeadowApp
+
 open System
 open Meadow.Devices
 open Meadow
 open Meadow.Foundation.Leds
 open Meadow.Foundation
+open Meadow.Peripherals.Leds
 
 type MeadowApp() =
-    // Change F7FeatherV2 to F7FeatherV1 for V1.x boards
-    inherit App<F7FeatherV2, MeadowApp>()
-        
-    do Console.WriteLine "Initialize hardware... [F#]"
-    let led = new RgbPwmLed(MeadowApp.Device, MeadowApp.Device.Pins.OnboardLedRed,MeadowApp.Device.Pins.OnboardLedGreen, MeadowApp.Device.Pins.OnboardLedBlue,Meadow.Peripherals.Leds.IRgbLed.CommonType.CommonAnode)
-    
-    let ShowColorPulse (color : Color) (duration : TimeSpan)  = 
+    // Change F7MicroV2 to F7Micro for V1.x boards
+    inherit App<F7FeatherV2>()
+
+    let mutable led : RgbPwmLed = 
+        null
+
+    let ShowcolorPulse (color : Color) (duration : TimeSpan) = 

```

Comment:
I'm not a great candidate for F# review, but this casing change didn't seem intentional.

```suggestion
    let ShowColorPulse (color : Color) (duration : TimeSpan) = 
```

Reply:
It was - the C should be uppercase